src/lib/objects/payggy/standalone.svelte

```diff
@@ -54,8 +54,12 @@
 	let toUser =
 		args.users.find((user) => user.address !== args.profile.address) || throwError('invalid user')
 	$: if (amount && token) {
-		const tokenToTransfer = { ...token, amount: toBigInt(amount, token.decimals) }
-		args.estimateTransaction(toUser.address, tokenToTransfer).then((f) => (fee = f))
+		try {
+			const tokenToTransfer = { ...token, amount: toBigInt(amount, token.decimals) }
+			args.estimateTransaction(toUser.address, tokenToTransfer).then((f) => (fee = f))
+		} catch (e) {
+			console.log({ e })
+		}
```

Comment:
I agree we can do this for now, but the solution is to have `amountString` and `amountBigInt` which gets parsed here and when there is error, we need to set the `fee` to `0` and the `amountBigInt` to `0` as well.

Reply:
I can also imagine that the input does not allow entering an invalid value. I tried to make the input accept the `type` passed in as prop and use a numeric input but unfortunately svelte does not allow this when the value is bound.